scripts/inspect_ckpt.py

```
@@ -0,0 +1,229 @@
+
+    # Sample weight diffs: load one from each, compute diff, free both
+    H("10b. BASE vs SFT sample weight diffs (serialized)")
+    diff_keys = [
+        "decoder.proj_in.weight", "decoder.proj_in.bias",
+        "decoder.proj_out.weight", "decoder.proj_out.bias",
+        "decoder.output_scale_shift_table", "null_condition_emb",
+    ]
+    for li in [0, 11, 23]:
+        for sub in ["self_attn.q_proj.weight", "self_attn.out_proj.weight",
+                     "cross_attn.q_proj.weight", "mlp.fc1.weight"]:
+            diff_keys.append(f"decoder.layers.{li}.{sub}")
```

Ruff (0.15.7)

[warning] Use `list.extend` to create a transformed list

(PERF401)